client/Components/App.js

```diff
@@ -1,30 +1,42 @@
+// React imports
 import React, { Component } from 'react'
 import { HashRouter as Router, Route, Switch } from 'react-router-dom'
-import { Home, Login, CodeEditor, AdminHomeView, SingleCohort } from './index'
-import { checkIfUserLoggedInThunk, fetchStretches } from '../store/thunks'
+
+// Redux imports
 import { connect } from 'react-redux'
+import { checkIfUserLoggedInThunk } from '../store/auth/actions'
+
+// For testing purposes only, remove as needed
```

Comment:
Is it worth creating an actions index file so that we can export and import all of the actions as one object?